R/pctr.R

```diff
@@ -52,7 +52,12 @@ pctr_at_product_level <- function(companies,
     rename(tilt_sector = "sec") |>
     pctr_add_scores(low_threshold, high_threshold)
 
-  left_join(companies, scored, by = "activity_uuid_product_uuid")
+  left_join(
+    companies,
+    scored,
+    by = "activity_uuid_product_uuid",
+    relationship = "many-to-many"
+  )
 }
```

Comment:
@maurolepore why the parameter `relationship = "many-to-many"` is introduced? I guess the `left_join` function will automatically do a `many-to-many` join. right?

Reply:
It avoids this warning:

```r
Warning (test-pstr_at_product_level.R:11:3): added column returns acceptable values
Detected an unexpected many-to-many relationship between `x` and `y`.
i Row 1 of `x` matches multiple rows in `y`.
i Row 49 of `y` matches multiple rows in `x`.
i If a many-to-many relationship is expected, set `relationship = "many-to-many"` to silence this warning.
```